src/components/common/RegAddEditOrgPerson.vue

```diff
@@ -281,19 +357,21 @@
             </div>
 
             <!-- Action Buttons -->
-            <div class="form__btns mt-10">
+            <div class="form__btns mt-6">
               <v-btn
+                id="btn-remove"
                 large outlined color="error"
-                class="btn-outlined-error btn-remove"
```

Comment:
If multiple buttons with the same name can be displayed simultaneously, then it's better use a classname to identify them (since duplicate IDs are an error).

Otherwise it's better to identify buttons with an ID.